README.md

```diff
@@ -19,7 +19,7 @@ the monitoring module.
 
 ## Documentation
 
-* [Installation](doc/02-Installation.md)
-* [Configuration](doc/03-Configuration.md)
-* [Templates](doc/04-Templates.md)
-* [Development](doc/06-Development.md)
+* [Installation](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/02-Installation/)
+* [Configuration](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/03-Configuration.md)
+* [Templates](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/04-Templates.md)
+* [Development](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/06-Development.md)
```

Comment:
Have you tried these links by chance? I guess not.

Reply:
Isn’t it enough for them to work after release?

Reply:
Come on, they are simply wrong:

https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/04-Templates/

Reply:
The work fine already: https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/03-Configuration/